utilities.py

```diff
@@ -751,6 +910,32 @@ def build_plots(
         include_plotlyjs = 'cdn',
     )
 
+    # To creat cal code distributions scatter
+
+    # Get list of all board ids
+    board_ids = sorted(df["data_board_id"].unique())
+    range_cal = [0, 1024]
+
+    # Copied pivot_df from build_time_plots
+    # Create the pivot table with a column for each board
+    pivot_df = df.pivot(
+        index = 'event',
+        columns = 'data_board_id',
+        values = list(set(df.columns) - {'data_board_id', 'event'}),
+    )
+    # Rename columns so they are no longer hierarchical
+    pivot_df.columns = ["{}_{}".format(x, y) for x, y in pivot_df.columns]
```

Comment:
Move this to the start of the function, after the "accepted" logic. Similar to the build_time_plots function